src/gui/widgets/TrackContentWidget.cpp

```diff
@@ -572,8 +572,7 @@ void TrackContentWidget::mousePressEvent( QMouseEvent * me )
 			so.at( i )->setSelected( false);
 		}
 		getTrack()->addJournalCheckPoint();
-		const TimePos pos = getPosition( me->x() ).getBar() *
-						TimePos::ticksPerBar();
+		const TimePos pos = getPosition( me->x() ).getTicks();
```

Comment:
Please revert this as this doesn't apply to BB. I deleted my old comment because it was wrong.

Reply:
I'm not sure I follow: the bug line 575 fixes is described above and I believe the solution is correct.

Reply:
This only brings a regression: clips are not created at the bar boundary now.

Reply:
Again, I'm not sure I'm understanding your point correctly, but in my testing the line does indeed fix the bug, so I believe it should be kept.

Reply:
I can't find any reason this should work. Could you try reverting this and reproducing the bug?

Reply:
This `if` statement covers the case where you add new `TrackContentObject`s(clips) in the song editor.

Reply:
> I can't find any reason this should work. Could you try reverting this and reproducing the bug?

I see, can't seem to be able to trigger the bug either, so that means you're right.